mesheryctl/pkg/utils/platform.go

```diff
@@ -653,6 +653,24 @@ func GetRequiredPods(specifiedPods []string, availablePods []v1core.Pod) ([]stri
 	return requiredPods, nil
 }
 
+// GetRequiredPodsMap checks if the pods specified by the user is valid returns a map of the required pods
+func GetRequiredPodsMap(specifiedPods []string, availablePods []v1core.Pod) (map[string]string, error) {
+	requiredPodsMap := make(map[string]string)
+	var availablePodsName []string
+	for _, pod := range availablePods {
+		availablePodsName = append(availablePodsName, pod.GetName())
+	}
+	for _, sp := range specifiedPods {
+		if index := StringContainedInSlice(sp, availablePodsName); index != -1 {
+			requiredPodsMap[availablePodsName[index]] = availablePodsName[index]
+		} else {
+			return nil, fmt.Errorf("pod \"%s\" specified, does not exist in the 'meshery' namespace. Please  Run `mesheryctl system status` to view the available pods", sp)
```

Comment:
@alphaX86 have we hardcoded support only deploying Meshery to a namespace titled `meshery`?

Reply:
Ok. I have removed.